runtime/src/utils/mod.rs

```diff
@@ -162,7 +162,7 @@ pub fn create_pool<S: Spawner + Metrics>(
         .spawn_handler(move |thread| {
             context
                 .with_label("rayon-thread")
-                .spawn_blocking(move || thread.run());
+                .spawn_blocking(false, move |_| thread.run());
```

Comment:
In a similar vein I wonder if it's clearer to just introduce a new public function rather than an opaque boolean (or, e.g. a binary enum). `spawn_blocking` and `spawn_dedicated/spawn_dedicated_blocking`. Each could share a backing "inner" function that takes in `true/false` from either function (as you have now), but abstracts away the boolean from external use which makes things clearer for readers of the code

Reply:
I started with this and opted for this pattern after playing with the code.